module1/projects/futbol/evaluation.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: Futbol Evaluation
+length: 2 week
+type: project
+---
+
+_[Back to Futbol Home](./index)_
+
+For this project, you will have a live evaluation over Zoom with an instructor. We may ask you to walk through your code, answer specific questions about your design, or the process you took to arrive at your finished project.
+
+To prepare for the live evaluation, you and your group should be able to answer the questions in the "sample evaluation" below. If you have any  questions, please let your instructor know before the project submission deadline.
+
+
+## Sample evaluation walkthrough:
+
+[ ] Demonstration of functional completeness
+ * Display to the viewer the ability to start the program via the runner file, and demonstrate a few of the statistics that can be calculated.
+
+[ ] Technical quality and organization of the code
+ * At a high level (not line by line), describe how you broke out this application. What classes did you create? What is the responsibility of each class? Why did you choose to design your code in this way?
+ * Is there a design decision that you made that you're particularly proud of?
+ * Were there any later parts of the project that were made more difficult by your early design choices?
+
+[ ] Identifying areas of code that you decided to refactor
+ * How did you identify areas of code to refactor? At a high level, how did you go about refactoring?
+ * Are there other areas of your code that you still wish to refactor?
+
+[ ] Discussion of collaboration/version control
+ * What was your collaboration like in this group project? How was it different working in a larger group compared to pairs?
+ * Were there any tools that you used to make collaboration easier?
+
+[ ] Discussion of test coverage
+ * Run your test suite and open the coverage report from SimpleCov.
+ * Show examples of unit and integration tests that you wrote.
+ * What was it like using the Spec Harness for this project? How did it inform your design or testing choices?
```

Comment:
In addition to them running their test suite, should we also have them run the spec harness?

Reply:
Added

Reply:
thanks!